resources/js/components/Calendar.vue

```diff
@@ -88,10 +86,12 @@
             currentYear: currentDate.getFullYear(),
             currentMonth: currentDate.getMonth() 
         };
-      },
-      computed: {
+    },
+    computed: {
         dayMonthAndYear() {
-            return `${this.currentDay} ${this.months[this.currentMonth]} ${this.currentYear}`;
```

Comment:
fatto così andrebbe bene se io non avessi inizializzato la data come la data corrente giusto?
Perchè in quel caso sarebbe giusto ogni volta che la data viene calcolata fare l'emit.
Secondo me, adesso non serve più. Però parliamone